k2/csrc/aux_labels.cc

```diff
@@ -11,10 +11,10 @@
 #include <utility>
 #include <vector>
 
-#include "glog/logging.h"
 #include "k2/csrc/fsa.h"
 #include "k2/csrc/fsa_util.h"
 #include "k2/csrc/properties.h"
+#include "k2/csrc/util/logging.h"
```

Comment:
I guess it's better if we put file `logging.h` in `k2/csrc/cuda/logging.h`, as files in `k2/csrc/` will be deleleted finally. And you even don't need to update those files and leave them with `glog`? (If it can be compiled successfully).